package-lock.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Removing deadcode that caused me initial setup confusion. Thanks @davidrideout for unblocking me!